### ISSUE TITLE
Comment Background color should be set to #272727

### ISSUE THREAD

**Issue by hqcked** [zohnannor/dark-mode-gd#1]
![image](https://github.com/zohnannor/dark-mode-gd/assets/42199661/71e0c256-3d87-4c12-a66d-11d2c1f27c68)

**Comment by zohnannor** [comment]
@hqcked thx for reporting, but I don't think you're right?

Here's all the main changed screens: 2.1+patcher on the left, 2.2+mod on the right. You can see it's exactly matches _or_ adds to the old recoloring.

<details>
<summary>Click to see</summary>

![image](https://github.com/zohnannor/dark-mode-gd/assets/35764628/22b26ea2-62c4-40db-ab9c-9d9bafd0393d)
![image](https://github.com/zohnannor/dark-mode-gd/assets/35764628/cc35da4d-ef8a-4a08-881a-c0913a858fcc)
![image](https://github.com/zohnannor/dark-mode-gd/assets/35764628/2df04530-298f-4199-8b64-28ba8c5c4fc5)
![image](https://github.com/zohnannor/dark-mode-gd/assets/35764628/ec849504-06f4-44c2-9c38-1c97899b2f68)
![image](https://github.com/zohnannor/dark-mode-gd/assets/35764628/dfcdf6f5-8f95-41bf-ba54-8affd0956a22)
![image](https://github.com/zohnannor/dark-mode-gd/assets/35764628/0d2f1ea5-1830-46e9-9e04-20aa3dbfa852)
![image](https://github.com/zohnannor/dark-mode-gd/assets/35764628/0b812dcf-427a-47f5-bcbf-e6ecc4ab8873)


</details>

Also, there is no `#272727` on your screenshot either.

![image](https://github.com/zohnannor/dark-mode-gd/assets/35764628/a66ab157-ea89-408d-be5d-e8c40ac73640)

**Comment by hqcked** [comment]
i cant find any other screenshots or videos other than your images that has the comment color be that dark

also theres no #272727 in the image cause the color isnt actually whats in the settings 
![image](https://github.com/zohnannor/dark-mode-gd/assets/42199661/b4dad9b6-694f-4585-896e-55770e5bcbc6)

**Comment by zohnannor** [comment]
I guess the darkmode patcher was updated at some point and the colors changed? Anyway, you can edit the colors yourself, but the issue is: the comment background (rounded rectangle with darker color) is semi-transparent, so I guess try a bit darker colors for it

**Comment by zohnannor** [comment]
I got it now, you've entered 272727 and it became 242424 which matched the 2.1+patcher color on your screenshot. I don't really know what version of the patcher was more popular (I got mine from Hyperbola's Cryptos TP) and what color I should leave as the default. I guess I'll see if more people will report the same issue, then I'll change the default to 272727. For now, yeah, set it to 272727 manually.
